src/dagmc/DagMC.hpp

```diff
@@ -30,6 +29,10 @@ namespace moab {
 
 class CartVect;
 
+#define DAGMC_VERSION 3.0
```

Comment:
These are no longer included from the DagMCVersion header?

Reply:
Thats either a new one on me, or one I was never aware of. Should I just include DagMCVersion.hpp?